### PR TITLE
[FSDE-1415] Improved Stat component to support new portal theme designs

### PR DESCRIPTION
**Pull request by mattnickles**
https://ctl.atlassian.net/browse/FSDE-1415

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-137/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://86ef3adfd4d9e269ab50674ddf99201afe0efe54-chi-pr.enterprise.ctl.io

**Review by jllr — APPROVED** [review]

File: src/chi/components/stat/stat.scss [hunk 1]
```diff
@@ -2,254 +2,401 @@
...
-  min-width: $stat-min-width;
-  position: relative;
-  text-align: $stat-text-align;
-  text-decoration: none;
-  width: 100%;
-  word-wrap: break-word;
-
-  &:hover {
+  flex-wrap: nowrap;
+  gap: 1.5rem; // sass-lint:disable-line no-misspelled-properties
```
Comment (janibolkvadze) [inline src/chi/components/stat/stat.scss:8]: Is this IE11 compatible?

File: src/website/views/components/stat/_examples.pug [hunk 2]
```diff
@@ -594,128 +1094,138 @@ h2 Examples
...
-                <div class="chi-stat-submetric">
-                  <div class="chi-stat-submetric__value">16</div>
-                  <div class="chi-stat-submetric__title">Elevated</div>
+        <div class="chi-stat -compact">
+          <div class="chi-stat__item">
+            <div class="chi-stat__content">
+              <div class="chi-stat-metric">
+                <div class="chi-stat-metric__title">Utilization</div>
+                <div class="chi-label__help">
+                  <button class="chi-button -icon -xs -flat" id="example__portal-help-button" aria-label="Help" data-target="#example__portal-help-popover" data-position="bottom" aria-describedby="example__portal-help-popover">
```
Comment (janibolkvadze) [inline src/website/views/components/stat/_examples.pug:1116]: Looks like the <script> BUTTON IDs need to be updated with
id="example__`portal`-help-button"
id="example__`portal`-help-button2"
id="example__`portal`-help-button3"

Comment (janibolkvadze) [reply src/website/views/components/stat/_examples.pug:1116]: Maybe we can address this in another PR.

**Review by janibolkvadze — APPROVED** [review]

**Pull request merged**